src/components/cards/card-booking/index.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState } from 'react';
+import { Button } from '../../button/button';
+import { BodySecondary } from '../../typography/body/body2';
+import { Caption } from '../../typography/caption/caption';
+import { Label } from '../../typography/label/label';
+import { Price } from '../../typography/price/price';
+import { ContainerCard } from '../containerCard';
+import { Divider } from '../divider';
+import './index.css';
+import { colors } from '../../../typography/colors';
+import { constants } from '../../../typography';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faCircleCheck, faHeart as heartRegular } from '@fortawesome/free-regular-svg-icons';
+import { faShare, faHeart as heartSolid } from '@fortawesome/free-solid-svg-icons';
+import { H3 } from '../../typography/headline/h3';
+import { formatPrice } from '../../../price-formatter';
+import { faWhatsapp } from '@fortawesome/free-brands-svg-icons';
+
+library.add(heartRegular, heartSolid);
+
+interface BookingPrices {
+  pricetotal?: number;
+  priceCondominium: number;
+  priceIptu?: number;
+  priceServiceFee?: number;
+  priceGrossRent?: number;
+}
+
+interface BookingProps {
+  nameUser: string;
+  imageUser: React.ReactNode;
+  prices: BookingPrices;
+  broker: string;
+  checkedUser: boolean;
```

Comment:
poderia deixar nullable para não precisar a prop se ela for `false`
```suggestion
  checkedUser?: boolean;
```